### ISSUE TITLE
Per-site IIS config not using the webroot

### ISSUE THREAD

**Issue by bdw429s** [Bilal-S/iis2tomcat#64]
I brought this up in https://github.com/Bilal-S/iis2tomcat/issues/59 a few months back and would like to mention it again.  The previous ticket was tagged as an enhancement but then was also closed, so I'm not sure if there were intentions to work on it or not.

I'm working on a CommandBox/IIS project again where my client wants to run more than one CF instance on a single VM and front them with IIS sites where each IIS site points to the AJP port of a specific CF instance.  In this case the web root of all the sites is the exact same folder so the `<webroot>/BIN`  convention doesn't work here.  It really doesn't feel right telling my client that I need to create dummy web root folders just to get BonCode to play nice and then pull everything in with virtual directories.  It would be a huge help here is Boncode had a way to work around this by adding the Boncode configuration directly to the IIS site so multiple sites pointing to the same folder of code can connect to different AJP ports on my background instances.

**Comment by Bilal-S** [comment]
Brad,
I remember this. I had closed the issue because you had stated that you had found a work-around.
I believe I had discussed the possibility to more configuration options into one file rather then having concerns/settings for each site separated by site in BIN folder (the IIS standard mechanism to separate concerns) . I will review again but not sure when I can implement since I am travelling heavily in the next weeks.

**Comment by bdw429s** [comment]
Yes, I found a workaround out of necessity, but it never negated my original request :)  I would still love to see this fixed for good.  Even if you can't work on it right away doesn't matter, it wouldn't be done in time for me to use on this project anyway (which concludes at the end of the week), but I'm sure it will come up again and I'd love to not have to resort to using the fake folder and virtual directory approach a 3rd time if possible.
